php82-arm.Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM --platform=linux/arm64 php:8.2-fpm-alpine
+
+ONBUILD ARG __VAPOR_RUNTIME=
+ONBUILD RUN if [ -z "$__VAPOR_RUNTIME" ] ; then \
+    echo "No runtime provided. Please upgrade to the latest version of laravel/vapor-cli." ; \
+    exit 1 ; \
+    elif [ "$__VAPOR_RUNTIME" != "docker-arm" ] ; then \
+    echo "The provided runtime [$__VAPOR_RUNTIME] is not supported by the vapor:php82-arm base image." ; \
+    exit 1 ; \
+    fi
+
+RUN apk --update add \
+    wget \
+    curl \
+    build-base \
+    libmcrypt-dev \
+    libxml2-dev \
+    pcre-dev \
+    zlib-dev \
+    autoconf \
+    oniguruma-dev \
+    openssl \
+    openssl-dev \
+    freetype-dev \
+    libjpeg-turbo-dev \
+    jpeg-dev \
+    libpng-dev \
+    imagemagick-dev \
+    imagemagick \
+    postgresql-dev \
+    libzip-dev \
+    gettext-dev \
+    libxslt-dev \
+    libgcrypt-dev \
+    linux-headers &&\
```

Comment:
What's the reason of including this?

Reply:
<img width="962" alt="Screenshot 2023-04-13 at 13 49 56" src="https://user-images.githubusercontent.com/3438564/231763849-576fab58-3425-440d-bb5f-28606a5ea88d.png">

It's in place of the above which I added initially to get PHP8.2 to build. Adding `linux-headers` means we can install sockets with `docker-php-ext-install` as we have done on all other PHP versions.

I have had a PR drafted for standard PHP8.2 for a while: https://github.com/laravel/vapor-dockerfiles/pull/10